src/test/lib/DataTest.cpp

```diff
@@ -3178,3 +3176,195 @@ QuicTestConnectAndIdleForDestCidChange(
         }
     }
 }
+
+void
+QuicTestStreamReliableReset(
+    )
+{
+    #define BUFFER_SIZE 10000
+    #define RELIABLE_SIZE 5000
```

Comment:
Since these are used beyond the scope of just this function, let's move them out. Also, we should test even bigger values (larger than 64k), which means we need to dynamically allocate the buffers (not on the stack) because kernel will crash if the stack usage is that big.